websites/R/Radio Garden/presence.ts

```diff
@@ -2,18 +2,48 @@ const presence = new Presence({
   clientId: "687070418804408445" //The client ID of the Application created at https://discordapp.com/developers/applications
 });
 
-presence.on("UpdateData", async () => {
-  if (document.location.pathname.startsWith("/listen/")) {
-    const elapsed = Math.floor(Date.now() / 1000);
-    //var container = document.querySelector(".channel-list")
-    //console.log(document.querySelector("div.mod-active").innerText)
-    const presenceData: PresenceData = {
-      details: `${document.querySelector("div.mod-active").textContent}`,
-      state: `${document.querySelector(".location-info-location").textContent}`,
-      largeImageKey: "bigglobe",
-      startTimestamp: elapsed
-    };
+let details: HTMLInputElement,
+state: HTMLInputElement;
 
-    presence.setActivity(presenceData);
-  }
-});
+presence.on("UpdateData", async () => {
+	if(!document.title.includes("Explore live radio by rotating")){
+		const elapsed =  Math.floor(Date.now() / 1000);
+		const presenceData: PresenceData = {
```

Comment:
merge consts

Reply:
Actually changed presenceData to let, for now it fixes some obscure behaviour until I figure out how to account for that.

Reply:
no, change it to const, it still should work. and merge consts

Reply:
Changed, merged. It seems to break timer, for some reason? Ill make that work and push changes later